app/src/main/java/com/babylon/wallet/android/domain/RadixWalletException.kt

```diff
@@ -10,6 +10,21 @@ import rdx.works.profile.data.model.apppreferences.Radix
 import rdx.works.profile.domain.ProfileException
 
 sealed class RadixWalletException(cause: Throwable? = null) : Throwable(cause = cause) {
+
+    sealed class GatewayException(cause: Throwable? = null) : RadixWalletException(cause) {
+
+        data class ClientError(override val cause: Throwable? = null) : GatewayException(cause)
+        data class ServiceError(
```

Comment:
minor: I would rename the `ServiceError` to `HttpError` because with these two Client and Service errors it looks like the first one is for 400 – 499 errors and the second for 500 – 599 errors, but it's not.

Reply:
You are right. It might get consfusing.